client/question/question.js

```diff
@@ -624,7 +625,8 @@ tie.constant('RUNTIME_ERROR_FEEDBACK_MESSAGES', {
         "Unfortunately Python doesn't support directly assigning ",
         "characters in a string. If you need to do so, try slicing the ",
         "string and adding new characters instead of assigning them. ",
-        "If you need a refresher on slicing, check out the primer."
+        "If you need a refresher on slicing, check out the ",
```

Comment:
wait this should be whatever we fixed to not be about slicing.

Reply:
I don't understand. I think this is the fixed version?

Reply:
ah, you're right. Sorry, I got slicing and splicing confused; my mistake.